culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
```

Comment:
I would add some context around merging, since it's a GitHub term.  Maybe like:

> at least one other person before becoming official, or "merging," meaning that requested changes are integrated into the main codebase.

This has me thinking that we need an entire "Welcome to GitHub, here are some useful terms" section, but maybe a link (later in the document when it's appropriate) would do the trick - those that are scratching their heads would click, others would continue reading.

Reply:
Yeah, since we're using GH as source-of-truth for docs, and we want less-technically-experienced contributors to be able to help write docs, we need some guides for them.